opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -155,25 +155,33 @@ message MetricDescriptor {
   repeated string label_keys = 5;
 }
 
+// LabelValues is a list of label string values.
+message LabelValues {
```

Comment:
Do you need this extra message anymore? You can directly use repeated string

Reply:
Fixed.